Sources/NIOHTTP2/HTTP2Error.swift

```diff
@@ -61,7 +61,11 @@ public enum NIOHTTP2Errors {
 
     /// A stream state transition was attempted that was not valid.
     public struct BadStreamStateTransition: NIOHTTP2Error {
-        public init() { }
+        let fromState: NIOHTTP2StreamState
```

Comment:
As this is currently only internal, users outside this package can't see it. We have two options here: the first is that we could make this public, the second is that we can conform this type to `CustomStringConvertible` and have the state be visible when printing this error. To begin with, I think I'd start with conforming to `CustomStringConvertible`.

Reply:
I updated the code to conform to CustomStringConvertible.